METS-Programme/openmrs-module-ugandaemr-sync
# Pulling Data from ARTAccess To UgandaEMR

https://app.asana.com/0/1133167201254913/1200117590862827

How to Test This 
-Ensure you have some patients having same Clinic no like those from the API. A query for this can be found on the asana card
-Pull PR down to local and watch sync module
-set dhis2 uuid to wepUthscBkh 
-Reschedule the Receive ARTAccess Visit data Task to run in the shortest time possible 
-Check whether patient data has been synced
